lib/logstash/outputs/elasticsearch.rb

```diff
@@ -392,7 +392,21 @@ def event_action_tuple(event)
     params[:version] = event.sprintf(@version) if @version
     params[:version_type] = event.sprintf(@version_type) if @version_type
 
-    [action, params, event.to_hash]
+    EventActionTuple.new(action, params, event)
+  end
+
+  class EventActionTuple < Array # TODO: acting as an array for compatibility
```

Comment:
ugly but I did not want to do a proper refactoring (replacing the `Array` with a proper struct) at this point, 
given this is meant to be a quick fix ...